src/riscv_cpu.cpp

```diff
@@ -1887,7 +1901,7 @@ RISCVCPUState *riscv_cpu_init(RISCVMachine *machine, int hartid)
                  ((uint64_t)2 << MSTATUS_SXL_SHIFT) |
                  (3 << MSTATUS_MPP_SHIFT);
     s->plic_enable_irq = 0;
-    s->misa |= MCPUID_SUPER | MCPUID_USER | MCPUID_I | MCPUID_M | MCPUID_A | MCPUID_F | MCPUID_D;
+    s->misa |= MCPUID_SUPER | MCPUID_USER | MCPUID_I | MCPUID_M | MCPUID_A;// | MCPUID_F | MCPUID_D;
```

Comment:
You don't need to change this line. The last misa line is overwriting the value.